packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1322,3 +1322,8 @@ export const MEETING_PERMISSION_TOKEN_REFRESH_REASON = 'ttl-join';
 
 // constant for named media group type
 export const NAMED_MEDIA_GROUP_TYPE_AUDIO = 1;
+
+// constant for statsAnalyzer report.callingServiceType
+export enum CALLING_SERVICE_TYPE {
+  TYPE_LOCUS = 'LOCUS',
```

Comment:
Oh that's surprising. I tried running some sample code in TypeScript Playground, it seemed to work for me. What's the error message here?

I noticed a lot of other variables in this file use `const` instead of `enum`. Would doing that fix this issue? (As an aside, there seems to be a lot of debate in the TypeScript community about using enums vs consts, I don't have a strong opinion but seems like people in general prefer avoiding enums?)

And I guess another thing... I feel like we should include 'CUCM', 'WEBEXCALLING', and 'BROADWORKS' here, even if we're not using them, for the reason I described in my previous comment: it provides more context in terms of what these options mean.

Reply:
I would add about enums, that if you're not using CALLING_SERVICE_TYPE as a type, I agree that it's not so useful to keep it as enum. I prefer keeping enum as a way to utilize duplication for types and values, but mostly for types, so it then should be inside type file, not a constants